recipes/node.rb

```diff
@@ -241,6 +241,17 @@
       action %i(enable start)
       ignore_failure true
     end
+
+    ruby_block 'Enforce running NM_CONTROLLED on host (>= 3.6)' do
+      block do
+        f = Chef::Util::FileEdit.new("/etc/sysconfig/network-scripts/ifcfg-#{node['network']['default_interface']}")
+        f.insert_line_if_no_match(/^NM_CONTROLLED/, 'NM_CONTROLLED=yes')
```

Comment:
@IshentRas on Centos VMs provisioned on Azure VMs, I have `NM_CONTROLLED=no` in /etc/sysconfig/network-scripts/ifcfg-eth0 (and network-manager package was not installed by default) ... I was wondering why I had trouble to run the cookbook on such environments. On my other environments, I have no `NM_CONTROLLED` line in the interface ifcfg script, but the cookbook works well.

Reply:
Not having NM_CONTROLLED is equivalent of turning it on.
Thanks for your comment. Good catch. 
@jperville Shall we just remove the line if it exists instead?